power_grid_model_io_native_c/power_grid_model_io_native_c/CMakeLists.txt

```
@@ -4,7 +4,8 @@
 
 # C API library
 add_library(power_grid_model_io_native_c SHARED
-  "src/handle.cpp"
+"src/handle.cpp"
+"src/vnf_pgm_converter.cpp"
```

I believe indentation here is prefered.
```suggestion
"  src/handle.cpp"
"  src/vnf_pgm_converter.cpp"
```